examples/order-manager/order_processor/order_processor.bal

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerinax/nats;
+import order_processor.types;
+import ballerina/lang.value;
+import ballerina/log;
+
+configurable string LISTENING_SUBJECT = ?;
+configurable string PUBLISH_SUBJECT = ?;
+
+// Creates a NATS client with default configurations
+nats:Client natsClient = check new(nats:DEFAULT_URL);
```

Comment:
Does not have to be a module level varaible.

Reply:
If I take this inside `onMessage`, for every message a client will be created.

Reply:
What I meant was to make it an object level variable.